src/fcast.c

```diff
@@ -69,8 +68,8 @@ SEXP fcast(SEXP lhs, SEXP val, SEXP nrowArg, SEXP ncolArg, SEXP idxArg, SEXP fil
           SET_STRING_ELT(target, k, (thisidx == NA_INTEGER) ? STRING_ELT(thisfill, 0) : STRING_ELT(thiscol, thisidx-1));
         }
       }
-      break;
-    case VECSXP:
+    }  break;
```

Comment:
case statements only need `{` when there's a local variable defined within that case. Otherwise it won't compile.  So I've rounded on only putting `{` when they are needed to remind my eyes that the case has local variables.  It doesn't matter and I'll merge anyway.  Just explaining the background.

Reply:
gotcha, the rule makes sense, good to know